libraries/Microsoft.Bot.Configuration/Services/GenericService.cs

```diff
@@ -3,6 +3,7 @@
 
 namespace Microsoft.Bot.Configuration
 {
+    using System;
```

Comment:
Move the using statements outside the Namespace. 
Add XML Comments (summary, params) to all public methods.